src/Dialogs/Waypoint/WaypointList.cpp

```diff
@@ -314,11 +317,14 @@ WaypointListWidget::UpdateList()
   if (dialog_state.type_index == TypeFilter::LAST_USED)
     FillLastUsedList(items, LastUsedWaypoints::GetList(),
                      way_points);
-  else
+  else {
     FillList(items, way_points, location, last_heading,
-             dialog_state,
-             ordered_task, ordered_task_index);
-
+             dialog_state, ordered_task, ordered_task_index);
```

Comment:
Why this change?

Reply:
easier to read

Reply:
But this change is not relevant to this PR; it is confusing reviewers.